beginning
foo.vm:1
var 4;
    ^
SyntaxError: Unexpected number
    at new Script (vm.js:*)
    at createScript (vm.js:*)
    at Object.runInThisContext (vm.js:*)
    at Object.<anonymous> (*test*message*vm_display_syntax_error.js:*)
    at Module._compile (internal/modules/cjs/loader.js:*)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:*)
    at Module.load (internal/modules/cjs/loader.js:*)
    at Function.Module._load (internal/modules/cjs/loader.js:*)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:*)
    at internal/main/run_main_module.js:*:*
test.vm:1
var 5;
    ^
SyntaxError: Unexpected number
    at new Script (vm.js:*)
    at createScript (vm.js:*)
    at Object.runInThisContext (vm.js:*)
    at Object.<anonymous> (*test*message*vm_display_syntax_error.js:*)
    at Module._compile (internal/modules/cjs/loader.js:*)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:*)
    at Module.load (internal/modules/cjs/loader.js:*)
    at Function.Module._load (internal/modules/cjs/loader.js:*)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:*)
    at internal/main/run_main_module.js:*:*
